### PR TITLE
Feature/water-meter

### PR DESCRIPTION
**Pull request by voloshchukV**
fixed check status error, Implemented alert message while status checking

### CODE REVIEW

File: eventGenerator/waterMeter.go [hunk 1]
```diff
@@ -77,7 +50,7 @@ func GenerateWaterMeterEvent() {
 		fmt.Println("Could not convert to JSON")
 	}
 
-	fmt.Println("Random valie is ", randomConsumtion)
+	fmt.Println("Water meter random valie is ", randomConsumtion)
```
Comment (undead404) [inline eventGenerator/waterMeter.go]: Another typo? `value`

File: eventGenerator/waterMeter.go [hunk 2]
```diff
@@ -91,32 +64,47 @@ func GenerateWaterMeterEvent() {
...
+		floodAler()
+		turnStatusBack()
+		return errors.New("Flood sensor alert! ")
 
+	}
+	return nil
 }
 
-func FloodAler() {
+func floodAler() {
```
Comment (undead404) [inline eventGenerator/waterMeter.go]: fix the typo please

**Review by undead404 — CHANGES_REQUESTED** [review]

File: server/main.go [hunk 3]
```diff
@@ -8,12 +8,14 @@ import (
...
 	"school-project-2019/server/domain"
 	"school-project-2019/server/domain/devices"
-
 	"github.com/rs/cors"
+	"github.com/subosito/gotenv"
 	//"school-project-2019/server/storage"
 )
 
 func main() {
+	gotenv.Load()
```
Comment (polluxx) [inline server/main.go:17]: Did you added the env file?
If it should be created for all developers - please add in README, how to do it.

**Review by polluxx — CHANGES_REQUESTED** [review]
Fix the comment please

File: server/main.go (hunk 3 above)
Comment (undead404) [reply server/main.go:17]: I'd suggest adding `.env.example` file with dummy values but proper keys

**Review by undead404 — APPROVED** [review]

**Review by polluxx — APPROVED** [review]

**Pull request merged**